frontend/src/features/mapSideBar/SidebarContents/subforms/LandUsageForm.scss

```diff
@@ -1,4 +1,5 @@
 .parcel-usage {
+  max-width: 900px;
```

Comment:
Don't change the overall width of the parent when the classification is changed.